packages/card/CardThumbnail.jsx

```diff
@@ -1,8 +1,18 @@
 import React, { useState } from "react";
-import VideoPlay from "../SVG-library/VideoPlay";
+import { cnb } from "cnbuilder";
 import MediaRatio from "../media/MediaRatio";
 import Modal from "../modal/ModalWrapper";
 import EmbedVideo from "../media/EmbedVideo";
+import { FAIcon } from "../icons/FAIcon";
+
+const videoPlayClasses = {
+  small: "su-left-13 su-bottom-13 [&>svg]:su-text-[4rem]",
+  medium:
+    "su-left-13 su-bottom-13 md:su-left-27 md:su-bottom-27 [&>svg]:su-text-[4rem] [&>svg]:md:su-text-[6rem]",
+  large: "su-left-13 su-bottom-13 [&>svg]:su-text-[4rem]",
+  featured:
+    "su-left-13 su-bottom-13 md:su-left-27 md:su-bottom-27 [&>svg]:su-text-[4rem] [&>svg]:md:su-text-[6rem]",
+};
```

Comment:
I replaced the map with this like I did with the `Container` package. I found this easier to read and update. I tried to replicate what the classes do for each variant (but replace width/height with font size - which seems to be how FAIcon works). Also, for the "medium" variant, I added `su-left-13`, `su-bottom-13` and have it go to 27 on MD breakpoint which should make it consistent with other variants on mobile.